### ISSUE TITLE
Multiple import maps

### ISSUE THREAD

**Issue by yoavweiss** [mdn/mdn#588]
### Proposal

Import maps currently have to load before any ES module and there can only be a single import map per document. That makes them fragile and potentially slow to use in real-life scenarios: Any module that loads before them breaks the entire app, and in apps with many modules the become a large blocking resource, as the entire map for all possible modules needs to load first.

There's a [proposal](https://github.com/whatwg/html/pull/10528) to enable multiple import maps per document, by merging them in a consistent and deterministic way.

We need to document that as part of https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/importmap, once it lands.

### Browser support

Not yet

### Tasks

- [ ] Send a PR against the relevant MDN content

### Dependencies

- [ ]

### Are you willing to support this work?

Yes

**Comment by bsmth** [comment]
Thanks for reporting. This is a dupe of #622, so closing in favor of that one
